qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -94,6 +94,9 @@ class TwirlingOptions:
         twirled layer.
 
       Default: "active-accum".
+
+    .. image:: /source_images/twirling_strategy_options.png
+      :alt: Twirling strategy options diagram 
```

Comment:
Rather than using the alt tag for the name of the diagram, it should contain text that helps an impaired user understand what's going on.  It shouldn't describe the visual characteristics of the image itself but should instead convey the same meaning as the image.

I don't know what's happening in the image or I'd try to come up with a suggestion.  It should be along the lines of "This image illustrates how the "active-accum option differs from the "all" option.  When using active accum..."

Reply:
Updated the text in https://github.com/Qiskit/qiskit-ibm-runtime/pull/2056/commits/e21227957f6600fc70f8af3ae50a0737529e2ee3